src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,66 @@
+.calendar {
+  width: $width-calendar + 6px;
+

```

Comment:
add some paddings because during hover effect your first raw will go out the page